Formula/nettle.rb

```diff
@@ -4,6 +4,7 @@ class Nettle < Formula
   url "https://ftp.gnu.org/gnu/nettle/nettle-3.6.tar.gz"
   mirror "https://ftpmirror.gnu.org/nettle/nettle-3.6.tar.gz"
   sha256 "d24c0d0f2abffbc8f4f34dcf114b0f131ec3774895f3555922fe2f40f3d5e3f1"
+  license "GPL-2.0"
```

Comment:
I think this is `LGPL-3.0`

See https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYING.LESSERv3

Reply:
I believe this one is multi-licensed with `GPL-2.0`, `GPL-2.0` and `LGPL-3.0` -- see [here](https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYINGv2) and [here](https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYINGv3) and [here](https://git.lysator.liu.se/nettle/nettle/-/blob/master/COPYING.LESSERv3).

According to [ravenports](http://www.ravenports.com/catalog/bucket_4B/nettle/standard/) it is:
```
GPLv2+ , LGPL3+ (dual)
```

Reply:
For multi license you should make a comment with the actual licenses and pick one. It needs to be clear what to add once multi license support is ready.

Reply:
@SmillerDev do you have an example of the actual comment you would like, for consistency?

Reply:
I'm just making something up right now, but how about something like:
`//license ['a', 'b'] - awaiting https://github.com/Homebrew/brew/pull/7953`

Reply:
How about:
```
# license ["GPL-2.0", "GPL-3.0", "LGPL-3.0"] -- pending https://github.com/Homebrew/brew/pull/7953
```

Reply:
And if it is ok, could this PR be merged and I will then send through another one with the required updates for `nettle`, thanks.

Reply:
yeah, we can certainly iterate on this process.

Reply:
If you can apply the same change to your previous PRs, that would be more than welcome, we can batch modify the formulae once the `brew` got merged.

Reply:
Thanks @chenrui333.

Reply:
Thank you for your help! ❤️